schemas/telemetry/crash/crash.4.schema.json

```diff
@@ -316,6 +316,9 @@
             },
             "userPrefs": {
               "type": "object"
+            },
+            "sandbox": {
+              "type": "object"
```

Comment:
Since we know which properties will live in the "sandbox" section, we should probably validate them as well. For example, this could become:

```
"sandbox": {
  "type": "object",
  "properties": {
    "effectiveContentProcessLevel": {
      "type": ["integer", "null"],
      "minimum": 0
    }
  }
}
```

This tells the schema that _sandbox_ can contain a property called _effectiveContentProcessLevel_ which can be either null or an integer with a minimum value of 0 (assuming that's the correct minimum).

The same change must be done in the other files as well.